src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js

```diff
@@ -162,7 +162,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
             return result;
           },
           thumbSize: function thumbSize() {
-            return this.smallThumbs ? 160 : 240;
+            return this.smallThumbs ? 110 : 195;
```

Comment:
You can't change this size here I think because it is not a supported size for the media resizing. So will provide unresized thumbnails

Reply:
ok, then:
`return this.smallThumbs ? 100 : 240;`